<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrdersTable</class>
 <widget class="QWidget" name="OrdersTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>666</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>941</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="5" column="1" rowspan="2" colspan="2">
     <widget class="QPushButton" name="openPriceListButton">
      <property name="text">
       <string>Открыть прайс-лист</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>Найти по</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QPushButton" name="deletingBtton">
      <property name="text">
       <string>Удалить заказ</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QPushButton" name="addingOrderButton">
      <property name="text">
       <string>Добавить заказ</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="searchComboBox"/>
    </item>
    <item row="0" column="0" rowspan="8">
     <widget class="QTableWidget" name="ordersTable"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="sortOrdersButton">
      <property name="text">
       <string>Отсортировать по</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="sortingOrdersComboBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="changeOrderButton">
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="changingOrderComboBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
